### ISSUE TITLE
Missing SBN UMD DOIs (10.26007) are not being delivered to ADS

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/doi-service#429]
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

When I did a search for a SBN UMD DOI like 10.26007/t2em-6844 in our DOI Search, it works as expected. But when I search for the same DOI in ADS, it does not appear.

### 🕵️ Expected behavior

I expected the DOI to be in ADS.

### 📜 To Reproduce

1. Go to https://pds.nasa.gov/tools/doi/#/search/10.26007/t2em-6844 to see the DOI is in our search
2. Go to ADS and no results: https://ui.adsabs.harvard.edu/search/fq=%7B!type%3Daqp%20v%3D%24fq_database%7D&fq_database=(database%3Aastronomy%20OR%20database%3Aphysics)&q=10.26007%2Ft2em-6844&sort=date%20desc%2C%20bibcode%20desc&p_=0

### 🖥 Environment Info

Linux production machine

### 📚 Version of Software Used

latest running in production

### 🩺 Test Data / Additional context

DOIs that should be in both searches:

10.26007/t2em-6844 - New Horizons Alice Ultraviolet Imaging Spectrograph Jupiter Encounter Raw Data. Findable since 2020.
10.26007/qaan-f992 - Calibrated Images with Geometric Backplanes for the Didymos Reconnaissance and Asteroid Camera for OpNav (DRACO) instrument, Version 1.0, 2021
10.26007/y367-c978 - SOlar and Heliospheric Observatory (SOHO) Solar Wind ANisotropies (SWAN) Derived Cometary Water Production Rates Data Collection, V3.0, 2022
10.26007/xt7t-jh11 - Rosetta Orbiter Spectrometer for Ion and Neutral Analysis (ROSINA) Double Focusing Mass Spectrometer (DFMS) Time Series Abundances, Version 1.0, 2023

**Comment by tloubrieu-jpl** [comment]
@jordanpadams @alexdunnjpl @c-suh can we close this ticket ?

**Comment by jordanpadams** [comment]
@tloubrieu-jpl unfortunately no. There is a discrepancy between our DOI database and legacy registry we are trying to work out.

**Comment by alexdunnjpl** [comment]
@jordanpadams is the outstanding issue here that no reports have been delivered in the relevant period?

It should be a simple thing to trigger a report with all data from the past (say) eight months, provided that won't cause an issue with potential duplicate results.  If we know exactly when deliveries stopped and when we restarted them, we could be surgical about it.

**Comment by jordanpadams** [comment]
@alexdunnjpl no. the issue is on the operations side with DOI metadata being mismatched between systems. no longer a software issue. thanks for checking.

**Comment by jordanpadams** [comment]
Status: manually reviewing DOIs to identify mismatches in DOIs. 82 mismatches (at least)

**Comment by jordanpadams** [comment]
Moving to backlog. Will not be able to get to this for this sprint.
